src/app/App.tsx

```diff
@@ -1,22 +1,39 @@
-import React, { Fragment } from 'react'
+import React, { Suspense } from 'react'
+import { useSelector } from 'react-redux'
+import cx from 'classnames'
 import GFWAPI from '@globalfishingwatch/api-client'
 import useGFWLogin from '@globalfishingwatch/api-client/dist/react-hook'
-import './App.css'
-import Map from '../map/Map'
+import { selectSidebarQuery } from '../routes/routes.selectors'
+import styles from './App.module.css'
+import Loader from '../loaders/Loader'
```

Comment:
I don't like relative imports at all, what about using this ? https://create-react-app.dev/docs/importing-a-component/#absolute-imports

Reply:
Good idea 👍 
I'm also bothered by how messy imports look like in components, now that they also act as containers.

Any suggestion for that ? Maybe:
- add `index.ts` file at the root of every feature folder, so that one single line of import is needed for each feature?
- maybe use containers sometimes?

Reply:
> add index.ts file at the root of every feature folder, so that one single line of import is needed for each feature?

do you mean something like:
- export the entire slice in the .slice file
```js
export default slice
```

and export its reducers and actions from the index
- index.ts
```js
import slice from './timebar.slice'
export const { reducer, actions } = slice
export * from './timebar.selectors'
export { default } from './Timebar'
```

so then in store.js
```js
import { reducer as timebar } from './timebar'
```

and then in App.tsx
```js
import Timebar from '../timebar'
```

??

This clarify the entry point and helps in things like exporting only the reusable actions but it adds an extra file... is this what you had in mind?

> maybe use containers sometimes?

When do you think will they be needed? This is the kind of things I prefer to identify and have the same pattern in every component.

Reply:
> is this what you had in mind?

Yes exactly. It's a bit of extra work but Map.tsx as 19 lines of imports already, so maybe it's worth it.

> containers

Containers are mostly boilerplate, and I find that `useSelector` and `useDispatch` are working nicely... But they also make components fatter, so I'm not 120% sold. What do you think?